random_numbers/mersennetwister.F90

```diff
@@ -92,6 +92,8 @@
 !> @{
 module MersenneTwister_mod
 ! -------------------------------------------------------------
+  use platform_mod, only: r8_kind
+
```

Comment:
It could... but the value gets assigned to a 4-byte integer, so I'm pretty confused what the original author had in mind here.

Reply:
I've removed the `_8` in fc57559edc068ca6b1407e6fbba70662c047ad20... but I'm feeling a bit uneasy about it, since someone presumably put it there for a reason that's not clear to me.

Reply:
It seems that the `_8` was a workaround to make it compile with GNU.

I've replaced this constant, as well as several others, with hexadecimal representations in 1d3f3218d60018ba6c91725d330ab15d476faeaf.